modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -168,7 +174,7 @@ protected void configure() {
                             .map("healthy"::equals)
                             .orElse(false);
                     } catch (IOException e) {
-                        logger().error("Unable to parse response {}", response);
+                        logger().error("Unable to parse response: " + response, e);
```

Comment:
put back `{}` for response, `Throwables` can be the last argument to `error()` calls.